hoodie-client/src/main/java/com/uber/hoodie/index/bloom/HoodieBloomIndex.java

```diff
@@ -170,8 +173,9 @@ public HoodieBloomIndex(HoodieWriteConfig config) {
     if (config.getBloomIndexPruneByRanges()) {
       // we will just try exploding the input and then count to determine comparisons
       // FIX(vc): Only do sampling here and extrapolate?
-      fileToComparisons = explodeRecordRDDWithFileComparisons(partitionToFileInfo,
-          partitionRecordKeyPairRDD).mapToPair(t -> t._2()).countByKey();
+      fileToComparisons = explodeRecordRDDWithFileComparisons(partitionToFileInfo, partitionRecordKeyPairRDD)
```

Comment:
can you check if the IDE formatting is kicking in?
https://github.com/apache/incubator-hudi/tree/master/style 
If these formatting changes are from those, its all good.

Reply:
I checked, this was just me. I'll leave it as is and just update the mapToPair bit